scripts/kube-init.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+set -x
+
+function clean_exit(){
+    local error_code="$?"
+    local spawned=$(jobs -p)
+    if [ -n "$spawned" ]; then
+        kill $(jobs -p)
+    fi
+    return $error_code
+}
+
+trap "clean_exit" EXIT
+
+# Switch off SE-Linux
+setenforce 0
+
+# Install docker if needed
+path_to_executable=$(which docker)
+if [ -x "$path_to_executable" ] ; then
+    echo "Found Docker installation"
+else
+    curl -sSL https://get.docker.io | sudo bash
+fi
+docker --version
+
+# Get the latest stable version of kubernetes
+export K8S_VERSION=$(curl -sS https://storage.googleapis.com/kubernetes-release/release/stable.txt)
+echo "K8S_VERSION : ${K8S_VERSION}"
+
+echo "Starting docker service"
+sudo systemctl enable docker.service
```

Comment:
This is mainly meant for travis. we can look into supporting other platforms. As mentioned, you really don't need this script, as long as you have localhost:8080 proxied to a working k8s instance for the tests to run.